src/applications/personalization/dashboard/components/FormList.jsx

```diff
@@ -40,7 +40,7 @@ class FormList extends React.Component {
     return !hasVerifiedSavedForms ? null : (
       <div className="profile-section medium-12 columns">
         <h2 className="section-header">Your applications</h2>
-        {forms.map(form => (
+        {verifiedSavedForms.map(form => (
```

Comment:
Discussed this with Ben a few days ago. We should only be mapping over the forms after we filtered them on line 37